vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj

```diff
@@ -104,16 +104,25 @@
     <Link>
       <ModuleDefinitionFile>react-native-win32.x64.def</ModuleDefinitionFile>
     </Link>
+    <ClCompile>
```

Comment:
I'm probably missing somehting :) But to me it seems this covers all possible combinations,
If so, could we lift it perhaps into the `<ItemDefinitionGroup above that doesn't have a condition?